lib/viewvc.py

```diff
@@ -1954,15 +1951,16 @@ def make_time_string(date, cfg):
             else:
                 tz = -time.timezone
             if tz < 0:
-                tz = "-%02d:%02d" % (-tz // 3600, (-tz % 3600) // 60)
+                tz = f"{-tz // 3600:02d}:{(-tz % 3600) // 60:02d}"
+                tz = f"-{tz}"
```

Comment:
Variable name `tz` here is very confusing.  It is used both of for raw timezone offset value (int) and for its string representation (str). 

These 2 lines can be merged:
```                tz = f"-{-tz // 3600:02d}:{(-tz % 3600) // 60:02d}"```